core/scripts/vrfv2plus/testnet/util.go

```diff
@@ -0,0 +1,305 @@
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrfv2_wrapper"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/vrfv2_wrapper_consumer_example"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+)
+
+func deployBHS(e helpers.Environment) (blockhashStoreAddress common.Address) {
+	_, tx, _, err := blockhash_store.DeployBlockhashStore(e.Owner, e.Ec)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func deployBatchBHS(e helpers.Environment, bhsAddress common.Address) (batchBHSAddress common.Address) {
+	_, tx, _, err := batch_blockhash_store.DeployBatchBlockhashStore(e.Owner, e.Ec, bhsAddress)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func deployCoordinator(
+	e helpers.Environment,
+	linkAddress string,
+	bhsAddress string,
+	linkEthAddress string,
+) (coordinatorAddress common.Address) {
+	_, tx, _, err := vrf_coordinator_v2plus.DeployVRFCoordinatorV2Plus(
+		e.Owner,
+		e.Ec,
+		common.HexToAddress(bhsAddress))
+	helpers.PanicErr(err)
+	coordinatorAddress = helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+
+	// Set LINK and LINK ETH
+	coordinator, err := vrf_coordinator_v2plus.NewVRFCoordinatorV2Plus(coordinatorAddress, e.Ec)
+	helpers.PanicErr(err)
+
+	linkTx, err := coordinator.SetLINK(e.Owner, common.HexToAddress(linkAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, linkTx, e.ChainID)
+
+	linkETHTx, err := coordinator.SetLinkEthFeed(e.Owner, common.HexToAddress(linkEthAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, linkETHTx, e.ChainID)
+	return coordinatorAddress
+}
+
+func deployBatchCoordinatorV2(e helpers.Environment, coordinatorAddress common.Address) (batchCoordinatorAddress common.Address) {
+	_, tx, _, err := batch_vrf_coordinator_v2plus.DeployBatchVRFCoordinatorV2Plus(e.Owner, e.Ec, coordinatorAddress)
+	helpers.PanicErr(err)
+	return helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func eoaAddConsumerToSub(e helpers.Environment,
+	coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus, subID uint64, consumerAddress string) {
+	txadd, err := coordinator.AddConsumer(e.Owner, subID, common.HexToAddress(consumerAddress))
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, txadd, e.ChainID)
+}
+
+func eoaCreateSub(e helpers.Environment, coordinator vrf_coordinator_v2plus.VRFCoordinatorV2Plus) {
+	tx, err := coordinator.CreateSubscription(e.Owner)
+	helpers.PanicErr(err)
+	helpers.ConfirmTXMined(context.Background(), e.Ec, tx, e.ChainID)
+}
+
+func eoaDeployConsumer(e helpers.Environment,
+	coordinatorAddress string,
+	linkAddress string,
+	keyHash common.Hash,
+	nativePayment bool) (
+	consumerAddress common.Address,
+	consumer *vrf_v2plus_single_consumer.VRFV2PlusSingleConsumerExample) {
```

Comment:
Lets deploy the external sub owner example instead of this one.

Reply:
Fixed this